src/Umbraco.Web.UI.Client/src/views/components/editor/umb-editor-content-header.html

```diff
@@ -40,18 +40,18 @@
                     </ng-form>
 
                     <a ng-if="content.variants.length > 0 && hideChangeVariant !== true" class="umb-variant-switcher__toggle" href="" ng-click="vm.dropdownOpen = !vm.dropdownOpen" ng-class="{'--error': vm.errorsOnOtherVariants}">
-                        <span>{{vm.currentVariant.language.name}}</span>
+                        <span ng-bind="getVariantDisplayName(vm.currentVariant)"></span>
```

Comment:
we try not to use methods in angular binding where possible because it's not good for performance since angular will call this method on every digest. Not sure we can avoid it since if we one-way bind it we'll have to handle refreshing it somehow, but maybe thats a possibility?